receiver/apachedruidreceiver/README.md

```diff
@@ -0,0 +1,68 @@
+# Apache Druid Receiver
```

Comment:
We have a different readme template than the standard otel one, and I can't seem to find it. I believe Jon linked it to Dakota for m365, check with either of them to grab the link and see if this is structured as expected.